src/expression/embeddedDocs/function/algebra/simplify.js

```diff
@@ -17,3 +17,39 @@ export const simplifyDocs = {
     'derivative', 'parse', 'evaluate'
```

Comment:
Can you add refrences to `simplify.resolve` and `simplify.simplifyCore` in `simplify`?

Reply:
Will do, to their new top-level names.